tms/src/main/java/com/netcracker/edu/tms/service/MailServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.netcracker.edu.tms.service;
+
+import com.netcracker.edu.tms.model.Mail;
+import org.springframework.scheduling.concurrent.CustomizableThreadFactory;
+import org.springframework.stereotype.Service;
+
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+@Service
+public class MailServiceImpl implements MailService {
+
+    private final String usernameForSession = "dmitrybobryakov@gmail.com";
```

Comment:
lets move configuring of sender address and password to application.properites
and add there flag notification.enable = true/false
which is false by default